gstreamer-sdp/src/s_d_p_key.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2018 Sebastian Dröge <sebastian@centricular.com>
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use ffi;
+
+pub struct SDPKey(ffi::GstSDPKey);
```

Comment:
I wasn't sure if just wrapping the ffi definition is fine here or not. There were no methods defined in the  -sys repo so I figured this was fine?

Also, I tried to auto gen these definition (and others) but I ran into the same error:
```
thread 'main' panicked at 'Missing memory management functions for GstSdp.SDPKey', src/codegen/record.rs:61:9
```
I'm guessing that's part of why these definitions had to be manually written?

Reply:
Just wrapping the FFI definition is fine but you'll have to implement memory management correctly and also implement the relevant traits from glib-rs. I didn't check if you did that here yet, just saying :) I'll take a closer look later.

For the memory management error, yes that's one of the reasons why these have to be manually implemented.